src/openhmd.c

```diff
@@ -97,6 +98,18 @@ int OHMD_APIENTRY ohmd_ctx_probe(ohmd_context* ctx)
 	return ctx->list.num_devices;
 }
 
+const char* OHMD_APIENTRY ohmd_gets(ohmd_string_description type)
```

Comment:
So these two strings are also statically allocated.  So It's probably fine to leave as-is, right?